test/integration_test/extender_test.go

```diff
@@ -240,3 +246,114 @@ func pvcOwnershipTest(t *testing.T) {
 
 	destroyAndWait(t, ctxs)
 }
+
+func antihyperconvergenceTest(t *testing.T) {
+	ctxs, err := schedulerDriver.Schedule("antihyperconvergencetest",
+		scheduler.ScheduleOptions{
+			AppKeys: []string{"test-sv4-svc-repl1"},
+		})
+	require.NoError(t, err, "Error scheduling task")
+	require.Equal(t, 1, len(ctxs), "Only one task should have started")
+
+	logrus.Infof("Waiting for all Pods to come online")
+	err = schedulerDriver.WaitForRunning(ctxs[0], defaultWaitTimeout, defaultWaitInterval)
+	require.NoError(t, err, "Error waiting for pod to get to running state")
+
+	scheduledNodes, err := schedulerDriver.GetNodesForApp(ctxs[0])
+	require.NoError(t, err, "Error getting node for app")
+	require.Equal(t, 3, len(scheduledNodes), "App should be scheduled on one node")
```

Comment:
nit: app should be scheduled on 3 nodes ?

Reply:
typo. thanks for noticing.